pkg/auditd/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM alpine:edge AS mirror
```

Comment:
Building from source won't be reproducible either.

I don't see a way to do a reproducible build without essentially replicating what we have for main for this. In any case, per a talk with @ncopa I've started the discussion about main inclusion for audit: https://github.com/alpinelinux/aports/pull/1776